system_setup/server/controllers/wslconfbase.py

```diff
@@ -43,6 +47,32 @@ class WSLConfigurationBaseController(SubiquityController):
         'additionalProperties': False,
         }
 
+    def __init__(self, app):
+        super().__init__(app)
+
+        # load the config file
+        data = {}
+
+        if path.exists('/etc/wsl.conf'):
```

Comment:
See my previous comment on defaults.

Reply:
default is already defined in the `types.py`, if you run the server, you can see the default value when querying from the API.